### DISCUSSION TITLE
Help - With transfer account

### DISCUSSION THREAD

**Discussion by TorontoKayaks** [community/community#51285] [New to GitHub]
I made a request for my webdeveloper to transfer the code resource from his github to mine. When he sent the transfer the email from Github contains this message 

After accepting this repository transfer, you will lose access to:
  - Codeowners functionality.
  - Any existing wikis.
  - Pulse, Contributors, Community, Traffic, Commits, Code Frequency and Network on the Insights page.
  - Draft PRs.
  - Multiple assignees for issues and PRs.
  - Multiple PR reviewers.
  - Branch and tag protection rules.
  - Any existing Pages.
  
  Was this transfer completed correctly? Will I lose access to my website content/details? Should I accept this transfer request if my main intention is to gain control of my code resource so I can have ownership of my website and create the new changes I want.

**Comment by queenofcorgis** [comment]
Hi there - is this a private repository you're transferring? Check out our docs if you haven't already, I found this [one](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository) to be very comprehensive.

**Comment by SMelidoni** [comment]
Hi @TorontoKayaks 👋🏽

Based on the message you received from GitHub, accepting the repository transfer will result in you losing access to some features and functionalities related to the repository with all the things you mentioned from the message.

However, accepting the transfer will not affect the code or content of the repository itself. You will still have access to the code and all of its files and history.

If your main intention is to gain control of your code resource and have ownership of your website to create the changes you want, accepting the transfer request is the correct step to take. It is normal for some features and functionalities to be lost in the process, but these can be set up again later as needed.

Just make sure to review the transferred repository and its contents thoroughly after the transfer to ensure that everything was transferred correctly and nothing was lost. If you have any concerns or questions, you can always contact GitHub support for assistance.
